Assets/Scripts/Editor/ParallaxEffect/ParallaxTool.cs

```diff
@@ -301,7 +301,11 @@ private void CreateObj(string pName, Sprite pSprite, int pLayer, float pSpeed, b
         obj.transform.SetParent(layerParent.transform);
         obj.transform.tag = LAYER_TAG;
         SpriteRenderer renderer = obj.AddComponent<SpriteRenderer>();
-        ParallaxEffect parEffect = obj.AddComponent<ParallaxEffect>();
+        obj.AddComponent<ParallaxEffect>();
+
+        ParallaxEffect parEffect = obj.GetComponent<ParallaxEffect>();
+
+        Debug.Log((parEffect == null));
```

Comment:
more Debug.Logs in the parallax scripts, just what we needed ;^)

Reply:
Removed the debug.log